### PR TITLE
fixing multiple issues

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

Fixed:
- Broken JS on failed async. The issue was when Captcha was misconfigured.
- Fixed broken manual import due to mandatory fields missing that are removed by the integration.

### CODE REVIEW

File: src/Blocks/components/form/assets/form.js [hunk 1]
```diff
@@ -504,26 +504,42 @@ export class Form {
 
 		if (this.state.getStateCaptchaIsEnterprise()) {
 			grecaptcha.enterprise.ready(async () => {
-				await grecaptcha.enterprise.execute(siteKey, {action: actionName}).then((token) => {
-					this.setFormDataCaptcha({
-						token,
-						isEnterprise: true,
-						action: actionName,
+				try {
+					await grecaptcha.enterprise.execute(siteKey, {action: actionName}).then((token) => {
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/form/assets/form.js:508]: Either do `.then` or `await` but not both 😄 

So
a)
```js
 grecaptcha.enterprise.execute(siteKey, {action: actionName}).then((token) => { ... })
```

or
b)
```js
const token = await grecaptcha.enterprise.execute(siteKey, {action: actionName});
...
```

I recommend `async/await`, it's cleaner and easier to read.

File: src/Blocks/components/form/assets/form.js [hunk 2]
```diff
@@ -504,26 +504,42 @@ export class Form {
...
 			});
 		} else {
 			grecaptcha.ready(async () => {
-				await grecaptcha.execute(siteKey, {action: actionName}).then((token) => {
-					this.setFormDataCaptcha({
-						token,
-						isEnterprise: false,
-						action: actionName,
+				try {
+					await grecaptcha.execute(siteKey, {action: actionName}).then((token) => {
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/form/assets/form.js:528]: Same as up

File: src/Blocks/components/form/assets/utils.js [hunk 3]
```diff
@@ -769,6 +769,44 @@ export class Utils {
...
+
+		// Set global msg.
+		this.setGlobalMsg(
+			formId,
+			msg ?? this.state.getStateSettingsFormServerErrorMsg(),
+			'error'
+		);
+
+		// Reset timeout for after each submit.
+		if (typeof this.GLOBAL_MSG_TIMEOUT_ID === "number") {
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/form/assets/utils.js:798]:
```suggestion
		if (typeof this.GLOBAL_MSG_TIMEOUT_ID === 'number') {
```

File: src/Enqueue/Blocks/EnqueueBlocks.php [hunk 4]
```diff
@@ -238,6 +238,7 @@ public function enqueueBlockFrontendScript(string $hook): void
 		);
 		$output['formResetOnSuccess'] = !\apply_filters(SettingsDebug::FILTER_SETTINGS_IS_DEBUG_ACTIVE, SettingsDebug::SETTINGS_DEBUG_SKIP_RESET_KEY);
 		$output['formServerErrorMsg'] = \esc_html__('A server error occurred while submitting your form. Please try again.', 'eightshift-forms');
+		$output['formCaptchaErrorMsg'] = \esc_html__('We have encountered a ReCaptcha error. Please try again.', 'eightshift-forms');
```
Comment (goranalkovic-infinum) [inline src/Enqueue/Blocks/EnqueueBlocks.php:241]:
```suggestion
		$output['formCaptchaErrorMsg'] = \esc_html__('A ReCaptcha error has occured. Please try again.', 'eightshift-forms');
```

File: src/Labels/Labels.php [hunk 5]
```diff
@@ -408,6 +408,7 @@ private function getCaptchaLabels(): array
 			'captchaWrongAction' => \__('The Captcha response "action" is not valid.', 'eightshift-forms'),
 			'captchaIncorrectCaptchaSol' => \__('The Captcha keys are not valid. Please check your site and secret key configuration.', 'eightshift-forms'),
 			'captchaScoreSpam' => \__('The automated system detected this request as a potential spam request. Please try again.', 'eightshift-forms'),
+			'captchaBrowserError' => \__('The Captcha responded with and invalid error. Please try again.', 'eightshift-forms'),
```
Comment (goranalkovic-infinum) [inline src/Labels/Labels.php:411]:
```suggestion
			'captchaBrowserError' => \__('The Captcha check encountered an error. Please try again.', 'eightshift-forms'),
```

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by iobrado — CHANGES_REQUESTED** [review]
One minor typo

File: src/Blocks/components/form/assets/form.js (hunk 1 above)
Comment (iruzevic) [reply src/Blocks/components/form/assets/form.js:508]: Added.

File: src/Blocks/components/form/assets/form.js (hunk 2 above)
Comment (iruzevic) [reply src/Blocks/components/form/assets/form.js:528]: Added.

**Review by iobrado — APPROVED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by kancijan — APPROVED** [review]

**Review by piqusy — APPROVED** [review]

**Pull request merged**